lib/flight_scheduler/cli.rb

```diff
@@ -67,7 +67,18 @@ def self.create_command(name, args_str = '')
     end
 
     create_command 'info' do |c|
-      c.summary = 'List the available partitions'
+      c.summary = 'List the available partitions and nodes'
+      c.slop.string '-O', '--format', <<~DESC
```

Comment:
Slurm has both a `-o`/`--format` and a `-O`/`--Format` option.  They are intended to do the same thing, but have different mechanics to achieve that.  I've requested that we copy the `-O`/`--Format` option as I think it is the easier to use and doesn't suffer from the limitations of the other.

What you've implemented is correct (pending some comments below).  However, you've gone with a `-O`/`--format` flag pair which doesn't seem quite right.  I think we should either use `-o`/`--format` or `-O`/`--Format`.  I'm not sure what I prefer here.  Lower case looks nicer, but consistency with slurm is good too.

Would you like to offer a suggestion here?